suite/suite.go

```diff
@@ -17,6 +19,7 @@ import (
 
 var allTestsFilter = func(_, _ string) (bool, error) { return true, nil }
 var matchMethod = flag.String("testify.m", "", "regular expression to select tests of the testify suite to run")
+var shuffle = flag.String("testify.shuffle", "off", "randomize the execution order of tests")
```

Comment:
Add a reference to `-test.shuffle`:

`randomize the execution order of tests in testify suites, like -test.shuffle`